### ISSUE TITLE
does these Web API work for external users for Power Apps Portal? 

### ISSUE THREAD

**Issue by aarohbits** [MicrosoftDocs/powerapps-docs#1494]
[Enter feedback here]

does these Web API work for external users for Power Apps Portal? 
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d2178fb4-0115-6e65-d13c-7dbf1365ceec
* Version Independent ID: 2bd841f9-64cc-c12a-7938-5748735c8256
* Content: [Client-side JavaScript using Web API in model-driven apps - Power Apps](https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/get-started-web-api-client-side-javascript)
* Content Source: [powerapps-docs/developer/common-data-service/webapi/get-started-web-api-client-side-javascript.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/developer/common-data-service/webapi/get-started-web-api-client-side-javascript.md)
* Service: **powerapps**
* GitHub Login: @JimDaly
* Microsoft Alias: **pehecke**

**Comment by aarohbits** [comment]
Thank you so much, Jim  for your clarification

**Comment by JimDaly** [comment]
No. They have to be a licensed user with privileges to perform actions on the CDS database.
They have to be Azure Active Directory users.
I believe the Portals app runs as a service user. I don't believe these APIs are exposed in Portal UI. It is a separate app. These APIs are available in Model-driven apps. Don't expect they exist in a public facing portal app.

**Comment by aarohbits** [comment]
Thanks, Jim for your reply.
